nilearn/datasets/func.py

```diff
@@ -398,8 +398,8 @@ def fetch_adhd(n_subjects=None, data_dir=None, url=None, resume=True,
     Parameters
     ----------
     n_subjects: int, optional
-        The number of subjects to load. If None is given, all the
-        40 subjects are used.
+        The number of subjects to load from maximum of 40 subjects.
+        If default None, only 30 subjects will be loaded.
```

Comment:
> We want smaller datasets for the tutorials.

The original issue #897 talked about CircleCI, this is why I was puzzled.

> If during a tutorial a user just types this function with no arguments, we want to download only 30 subjects, as it's enough for most things.

What is puzzling me here is: Is nilearn a tool made for tutorials or for studies? People that wants to actually study this dataset may be puzzled by this behavior.

I would be in favor of creating a `fetch_tutorial_adhd` for this special purpose. Because, if space is the only issue, I suggest to restrain the study to the lighter subjects. Here are some stats:

Dataset | Control/ADHD count | Size in MB
----------|----------|------------
Full ADHD | 20/20 | 1791.88
30 first subjects of ADHD | 15/15 | 1305.22
30 smaller subjects of ADHD | 16/14 | 1128.36
24 smaller subjects of ADHD | 13/11 | 785.87
15 smaller subjects of ADHD | 8/7 | 398.28